tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java

```diff
@@ -226,87 +256,124 @@ private static String toJsonString(Map<String, Object> data) {
         return json;
     }
 
-    private static String statusAsJson(long consumed, long remaining, String transactionalId) {
+    private static synchronized String statusAsJson(long totalProcessed, long consumedSinceLastRebalanced, long remaining, String transactionalId, String stage) {
```

Comment:
Need to add synchronized as we are using static date `FORMAT` which shall triggers spotsbug for concurrent access.